pom.xml

```diff
@@ -52,6 +52,7 @@
 		<module>zuul-config-discovery</module>
 		<module>zuul-proxy</module>
 		<module>zuul-proxy-eureka</module>
+		<module>tomcat-7.x</module>
```

Comment:
I figured if there were other tomact 7 tests we wanted to add in the future we could just have one project for them.  If you think that would not be the case I am fine renaming it.